akka-docs/src/main/paradox/general/stream/stream-design.md

```diff
@@ -45,6 +45,37 @@ All stream Processors produced by the default materialization of Akka Streams ar
 
 This means that `Sink.asPublisher(true)` (for enabling fan-out support) must be used where broadcast behavior is needed for interoperation with other Reactive Streams implementations.
 
+### Rationale and benefits from Sink/Source/Flow not directly extending Reactive Streams interfaces
+
+An often overlooked fact about [Reactive Streams](https://github.com/reactive-streams/reactive-streams-jvm/) is that they are a [Service Provider Interface](https://en.m.wikipedia.org/wiki/Service_provider_interface).
+This is confirmed even in the [earliest discussions](https://github.com/reactive-streams/reactive-streams-jvm/pull/25) about the specification dating back to 2014.
+Akka Streams was designed along with the development of Reactive Streams, with heavy involvement of the Akka team, as such, we understand the purpose of these interfaces very well.
+
+In the earliest days of the specification even the spec's types attempted to hide Publisher, Subscriber and the other types.
+Sadly the types would leak regardless in the back then considered API, [thus the API(!) was removed and the SPI types are all that remained](https://github.com/reactive-streams/reactive-streams-jvm/pull/25).
+
+Nowadays you see some implementations of Reactive Streams claim that their direct extending of these types is a benefit for some reason.
+This is not correct, as such was not, and is not the goal of the Reactive Streams interfaces.
```

Comment:
`This is not correct` is not friendly.

Reply:
Propose a rewording? "it's not correct as such was not the goal" is a normal sentence to me